src/windows/ScrollView.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+* ScrollView.tsx
+*
+* Copyright (c) Microsoft Corporation. All rights reserved.
+* Licensed under the MIT license.
+*
+* RN Windows-specific implementation of the cross-platform ScrollView abstraction.
+*/
+
+import React = require('react');
+import RN = require('react-native');
+import {ScrollView as ScrollViewBase} from '../native-common/ScrollView';
+
+import EventHelpers from '../native-common/utils/EventHelpers';
+
+export class ScrollView extends ScrollViewBase {
+
+    protected _render(props: RN.ScrollViewProps): JSX.Element {
+
+        var onKeyDownCallback = this.props.onKeyPress ?
+                // We have a callback function, call the wrapper
+                this._onKeyDown :
+                undefined;
+
+        // TODO: #737970 Remove special case for UWP when this bug is fixed. The bug
+        //   causes you to have to click twice instead of once on some pieces of UI in
+        //   order for the UI to acknowledge your interaction.
+        const keyboardShouldPersistTaps = 'always';
+
+        return (
+
+            <RN.ScrollView
```

Comment:
extra new line

Reply:
ok

---
In reply to: [159358750](https://github.com/Microsoft/reactxp/pull/404#discussion_r159358750) [](ancestors = 159358750)